### DISCUSSION TITLE
Weird double green screen

### DISCUSSION THREAD

**Discussion by jpofsouza** [tsunamods-codes/7th-Heaven#230] [Q&A] [answered]
Every time I open FF7 from 7th Heaven the game opens with two distorted green screens. But if I open the game using Steam only, it opens normally. The game works normally except for the visuals.

![Screenshot 2024-10-17 215645](https://github.com/user-attachments/assets/567f13ec-d588-4e37-9b95-bea2b79df4e6)

My suspicion is the graphics API. I'm playing on an old notebook (Intel Core i3-370m) and its GPU supports at most OpenGL 2.0 and DirectX 10, which is enough for FF7 vanilla but apparently not for 7th Heaven. So, I just want to know if there really is such a minimum requirement.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Yes, 7th uses FFNx behind the scene which is meant for modern hardware. Your PC configuration unfortunately doesn't meet the specific requirements. Modding the game requires a modern PC ( and at a least a modern GPU ), since we do much more than just running the game vanilla. If you have a Steam Deck you can have a look at https://github.com/dotaxis/7thDeck where it runs perfectly fine.
